src/conmon.c

```diff
@@ -737,9 +782,15 @@ static bool read_stdio(int fd, stdpipe_t pipe, gboolean *eof) {
     nwarnf("stdio_input read failed %s", strerror(errno));
     return false;
   } else {
-    if (write_k8s_log(log_fd, pipe, buf, num_read) < 0) {
-      nwarn("write_k8s_log failed");
-      return G_SOURCE_CONTINUE;
+    // Branch on log driver type to find where to write to
+    if (g_log_driver == JOURNALD) {
+      sd_journal_print(5, buf);
```

Comment:
this is unsafe.

First we need to make sure "buf" is zero terminated.  This can be easily done by increasing the buffer size by one, and writing `buf[num_read] = '\0';` on success

Also please use sd_journal_print(5, "%s", ..), as we don't want to feed directly in the formatter string.